zerologr.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/rs/zerolog"
 )
 
+const (
+	defaultLevelDelta zerolog.Level = 1
```

Comment:
Can you use this magic variable to explain how zerolog handles log levels and how that maps to logr levels?

Reply:
Borrowed from here: https://github.com/bombsimon/logrusr/blob/d198d3e84a78c6115fb1e18c1ccb66193530e9c7/logrusr.go#L18.  No magic, but it's an explicitly-named control knob instead of a magic number.  It could be offered as a parameter to the user if they want to shift the default level assignments (I could actually see myself doing this given the limited number of logging levels in zerolog).  But I didn't offer a way to do that, so no magic yet.